aten/src/ATen/native/mps/operations/ReduceOps.mm

```diff
@@ -771,7 +770,7 @@ Tensor std_var_common_impl_mps(
                                                                      name:nil];
           MPSGraphTensor *outputTensor;
 
-          if (use_correction && correction_value)
+          if (correction_value)
```

Comment:
The default for `correction_value` is 1, is this condition works as intended?

Reply:
Yes, that's the crux of a bugfix: default value for `unbiased` is True rather than false.

Reply:
Still bothers me a bit that now `correction_value` is an integer and it's not clear from the condition here and before.
It can only be 0 or 1?

Reply:
Nevermind, it was integer even before, `false` in `const auto correction_value = use_correction ? correction.value() : false;` was misleading.

Reply:
Exactly, one of the opinfo test even passes `torch.var(x, correlation=2)` and compares the CPU output.